### DISCUSSION TITLE
FATA[0000] Argo CD server address unspecified

### DISCUSSION THREAD

**Discussion by ruslan-byondxr** [argoproj/argo-cd#7073] [Q&A]
The argocd UI running and working. 
Im trying to list apps with 
```shell 
argocd app list 
 ```
 
or

```shell 
argocd app get svc-cart
 ```

Thats what I get :
```shell
FATA[0000] Argo CD server address unspecified
```

**Version**

```shell
{
    "Version": "v2.1.0+d0b2d55",
    "BuildDate": "2021-08-20T05:30:54Z",
    "GitCommit": "d0b2d55e3fb7fe8b17385d6687886de41651f31b",
    "GitTreeState": "clean",
    "GoVersion": "go1.16.5",
    "Compiler": "gc",
    "Platform": "linux/amd64",
    "KsonnetVersion": "v0.13.1",
    "KustomizeVersion": "v4.2.0 2021-06-30T22:49:26Z",
    "HelmVersion": "v3.6.0+g7f2df64",
    "KubectlVersion": "v0.21.0",
    "JsonnetVersion": "v0.17.0"
}
```

**Logs**

```shell
FATA[0000] Argo CD server address unspecified
```

**Comment by alexmt** [comment]
Hello @ruslan-byondxr ,

You need to specify server host using `--server` flag or run `argocd login <server-host>`. E.g.
* `argocd login cd.apps.argoproj.io` 
* `argocd app list --server cd.apps.argoproj.io`

**Comment by ruslan-byondxr** [comment]
@alexmt 
Sorry, i don't understand what you mean.
I'm running argocd locally. Which server you are talking about ?

**Comment by alexmt** [comment]
The `argocd` can talk to different argocd instances. So you need to specify server address using `argocd login` command or `--server` flag.

Are you running argocd server locally, as a process? In this case you can use `argocd login localhost:8080`.

If argocd is installed in minikube then you can use `argocd login --core` - in this mode argocd talks directly to k8s cluster and don't need api server at all.

**Comment by ruslan-byondxr** [comment]
@alexmt 

Running 
```argocd login --core``` 

Getting:
`Error: unknown flag: --core`

**Comment by 123MwanjeMike** [comment]
Simply run with the `--server` flag.
For example if ArgoCD server is accessible on argocd.domain.com

Run,
`argocd app get svc-cart --server=argocd.domain.com`

**Comment by Shivappa** [comment]
The error itself states to mention the server address.
So, you can give domain or cluster IP address using below command.
`argocd login <ClusterIPAddress>:<port>`
Ex: `argocd login 10.102.1.4:4615`

Then it asks for `username` and `password`, input the credentials.
Execute `argocd app list` now it shows all the apps and their status.
